bootstrap-gruntwork-installer.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+#
+# A bootstrap script to install the Gruntwork Installer.
+#
+# Why:
+#
+# The goal of the Gruntwork Installer is to make make installing Gruntwork Script Modules feel as easy as installing a
+# package using apt-get, brew, or yum. However, something has to install the Gruntwork Installer first. One option is
+# for each Gruntwork client to do so manually, which would basically entail copying and pasting all the code below.
+# This is tedious and would give us no good way to push updates to this bootstrap script.
+#
+# So instead, we recommend that clients use this tiny bootstrap script as a one-liner:
+#
+# curl -Ls https://raw.githubusercontent.com/gruntwork-io/gruntwork-installer/master/bootstrap-gruntwork-installer.sh | bash -s --version=0.0.12
+#
+# You can copy this one-liner into your Packer and Docker templates and immediately after, start using the
+# gruntwork-install command.
+
+set -e
+
+readonly DEFAULT_FETCH_VERSION="v0.0.3"
+readonly FETCH_DOWNLOAD_URL_BASE="https://github.com/gruntwork-io/fetch/releases/download"
+readonly FETCH_INSTALL_PATH="/usr/local/bin/fetch"
```

Comment:
Just calling out that clients who use the version of `fetch` installed by our installer may write scripts that rely on that version of fetch. Are we better off calling this `/usr/local/bin/gruntwork-fetch` so they are free to install whatever version fetch they want?